docs/ways-you-can-help-us.md

```diff
@@ -66,6 +66,33 @@ open-source firmware mission. It is straightforward if you don't have time for
 other contributions, and there are multiple options - feel free to choose one
 that works for you.
 
+### Become a Dasharo Supporter
+
+Buy the Dasharo Supporters Entrance subscription if you own/use/buy a machine
+running Dasharo firmware:
+
+* [1-year subcription](https://3mdeb.com/shop/open-source-hardware/subscription-for-dasharo-compatible-with-msi-pro-z-690a-ddr4/)
+* [2-year subscription](https://3mdeb.com/shop/open-source-hardware/2-year-dasharo-supporters-entrance-for-msi-pro-z690-a-ddr4/)
+* [3-year subscription](https://3mdeb.com/shop/open-source-hardware/3-year-dasharo-supporters-entrance-for-msi-pro-z690-a-ddr4/)
+
+Or buy the hardware including the subscriptions from us, e.g.:
+
+* [MSI PRO Z690-A DDR4 motherboard with Dasharo Supporters Entrance](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-motherboard/)
+* [MSI PRO Z690-A DDR4 with Dasharo Supporters Entrance – Full PC Build](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/)
+* or other platforms
+
+As a Dasharo Supporter you will get:
+
+* The latest Dasharo releases issued by Dasharo Team
+* Special Dasharo Updates for Supporters –  Community receives one update per
```

Comment:
> Community receives one update per year

Are we sure about that? Do we agree that publicly available update will be released one per year?

Reply:
I'm asking about this piece of text to make sure what kind of attitude we have towards the update now. The earlier plan was that initial deployment is publicly available and possibility to update from within the menu reserved for Dasharo SE.

I also understand that the last release required that the upgrade capability be made available to all users, and that's why we implemented the first version of the upgrade from the menu - I preferred to elaborate/describe in the comments to make sure we're on-the-same-page here.

Reply:
Right now the idea is precise as described by @miczyg1 [here](https://github.com/Dasharo/docs/pull/497/files). Please let me know is this is clear enough, if not we will try to improve the description.

Reply:
Now it is clear. To summarize. We want to release one community release per year that bumps up the minor version and up to two Dasharo SE releases (can be zero, one or two) that bumps up the patch version. The community release will have all the changes of the previously released Dasharo SE releases.